Makefile

```diff
@@ -37,6 +37,13 @@ install:
 	# Copy dracut and systemd config files
 	cp -Prf baseimage/dracut $(DESTDIR)$(prefix)/share/doc/bootc/baseimage/dracut
 	cp -Prf baseimage/systemd $(DESTDIR)$(prefix)/share/doc/bootc/baseimage/systemd
+	# Install bootc-destructive-cleanup in fedora derivatives 
+	@ID=$$(grep '^ID=' /usr/lib/os-release | cut -d'=' -f2 | tr -d '"'); \
```

Comment:
The os-release file was explicitly designed to be parsable by actually just `.` by a posix shell (i.e. sourcing it). See e.g. `OS_ID=$(. /usr/lib/os-release && echo $ID)` which we do in `installdeps.sh`